caravel/assets/javascripts/explorev2/actions/exploreActions.js

```diff
@@ -206,3 +208,11 @@ export function changeFilterOp(filter, op) {
 export function changeFilterValue(filter, value) {
   return { type: CHANGE_FILTER_VALUE, filter, value };
 }
+
+export function setPivotColumns(pivotColumns) {
+  return { type: SET_PIVOT_COLUMNS, pivotColumns };
```

Comment:
do you need to have pivotColumns and aggMetric separate actions ?
I believe a server returns the bootstrap data, in that case it would be more readable to have just a single action